### DISCUSSION TITLE
Extracting bundle

### DISCUSSION THREAD

**Discussion by jcasale** [wixtoolset/issues#7800] [Questions]
In wix v4, I am able to extract a bundle and obtain the embedded msi files, for example:

```
dark.exe VC_redist.x64.exe -x VC_redist.x64
```

However, using wix v4 doesn't extract the embedded files using their original names:

```
wix burn extract VC_redist.x64.exe -o VC_redist.x64
```

The first invocation extracts `VC_Runtime_arm64.msi` whereas the second extracts the same file as `a0`.

Is there anyway to achieve the same output in v4?

**Comment by robmen** [comment]
Probably need to update the v4 code to add the functionality back.
